.github/workflows/ci.yaml

```
@@ -14,12 +14,11 @@ jobs:
           - IMAGE: melodic
             CCOV: true
             CATKIN_LINT: true
-            UPSTREAM_WORKSPACE: 'upstream.rosinstall'
+            UPSTREAM_WORKSPACE: 'melodic_upstream.rosinstall'
```

We can get rid of the extra file (`upstream.rosinstall`) like this:
```suggestion
            UPSTREAM_WORKSPACE: github:PickNikRobotics/rviz_visual_tools#master
```

---

Thanks for the suggestion. Done.

---

You are welcome.